modules/graphics/src/font.cxx

```diff
@@ -147,6 +118,8 @@ namespace Moon {
       bbox.y = 0;
       bbox.z = 0;
       bbox.w = 0;
+    } else {
+      if (cursor > bbox.z) bbox.z = cursor;
```

Comment:
```c++
// Adjusts bbox's width based on the current cursor position, since the bbox may be a smidgen // smaller than the intended

// tl;dr the final bounding box forgot the glyph-advance of the last charatcer causing it to clip.

```